experiments/snippets/dragAndDropOpportunityWall/README.md

```diff
@@ -0,0 +1,18 @@
+# Opportunity Planning Wall
+
```

Comment:
On v1 at 12.2.6. No dice when I try to drop an opportunity in a new column. This page **does not appear to work**.

Reply:
Found other examples that work and replaced JS.